src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -7044,6 +7044,36 @@
     crafted: true
 """
 
+helps['network bastion ssh'] = """
+type: command
+short-summary: SSH to a virtual machine using Tunneling from Azure Bastion.
+examples:
+  - name: SSH to virtual machine using Azure Bastion (autogenerated)
+    text: |
+        az network bastion ssh --name MyBastionHost --resource-group MyResourceGroup --target-resource-id vmResourceId --auth-type AAD/password/ssh-key-file --username xyz
+    crafted: true
```

Comment:
Please add different examples for different `--auth-type`

Reply:
added